.github/workflows/misc.yml

```diff
@@ -48,6 +48,7 @@ jobs:
       - run: sudo -E bash scripts/environment/bootstrap-ubuntu-20.04.sh
       - run: bash scripts/environment/prepare.sh
       - run: echo "::add-matcher::.github/matchers/rust.json"
+      - run: sudo chmod 755 /var/lib/vector
```

Comment:
Nit, but could we add this to `scripts/environment/bootstrap-ubuntu-20.04.sh`? That would avoid us forgetting to add it to new workflows.

Reply:
Neat, good idea. Done in https://github.com/vectordotdev/vector/pull/19659/commits/cad4c0b3bc953f7253902410c7bf76d4dae30152